.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "features": {
+  }
+}
```

Comment:
what is this?

Reply:
You've made the PR, could you please explain what this dev container is for?

Reply:
Bisa bantu saya cara membuat apk untuk membaca pasan sms dari pelanggan yang mengunakan apk saya

Reply:
That is a service bandwidth offers, you need to reach out to a salesperson to gain access to our APIs. https://www.bandwidth.com/messaging/sms-api/